scss/components/_banner.scss

```diff
@@ -31,10 +44,11 @@
     }
 
     &__body {
-        display: inline-block;
+        display: flex;
```

Comment:
`display: flex` is only partially supported by IE11. It looks like we might need a `-ms` prefix, or look at other ways